helm/reana/README.md

```diff
@@ -35,6 +35,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_workflow_controller.imagePullPolicy`   | REANA-Workflow-Controller image pull policy                                          | IfNotPresent                                    |
 | `components.reana_workflow_controller.environment.REANA_JOB_HOSTPATH_MOUNTS`   | JSON list of optional hostPath mounts, for all user jobs. Each mount object has a key `name` (name of the mount), `hostPath` (path to the directory to be mounted from the Kubernetes nodes) and `mountPath` (path inside the job containers where the `hostPath` will be mounted)            | None                                            |
 | `components.reana_workflow_controller.environment.REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES` | Keep alive Kubernetes user runtime jobs depending on status (`finished` and/or `failed`). | None |
+| `components.reana_workflow_controller.environment.REANA_WORKFLOW_ENGINE_CONTAINERS_MEMORY_REQUESTS` | Base memory requested by each workflow engine container. For possible value formats see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | None |
```

Comment:
The variable might not be very clear, I would like to get some feedback.

Explanation: It governs the memory limits per container (see [implementation](https://github.com/reanahub/reana-workflow-controller/pull/367/files#diff-0941dd1fa9e589338659a84e2274936332c7498f29b6727873957b99fbb7b1fcR534)) since there is no such thing as limits for whole pods, therefore if one wants to set 1Gi limit and 512Mi request for workflow engines in total, one has to set:
```diff
     environment:
       SHARED_VOLUME_PATH: /var/reana
+      REANA_WORKFLOW_ENGINE_CONTAINERS_MEMORY_REQUESTS: 256Mi
+      REANA_WORKFLOW_ENGINE_CONTAINERS_MEMORY_LIMITS: 512Mi
   reana_workflow_engine_cwl:
```

Options:
1. Split config variable at container level:
```diff
- REANA_WORKFLOW_ENGINE_CONTAINERS_MEMORY_REQUESTS
+ REANA_WORKFLOW_ENGINE_CONTAINER_ENGINE_MEMORY_REQUESTS 
+ REANA_WORKFLOW_ENGINE_CONTAINER_JOB_CONTROLLER_MEMORY_REQUESTS 
```
2. Take the value entered by the admin and split it equally among the containers (easier to understand by an admin with a variable named like `REANA_WORKFLOW_ENGINE_MEMORY_REQUESTS`):
    - Note: we would need some kind of translation code, see [possible values for the field](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory) (e.g. `512Mi` if there are 2 containers, `"{value}{unit}".format(value=get_numerical_value("512Mi")/2, unit=get_unit_value("512Mi")`.
 3. Leave as it is, since the work in master (0.8) of splitting `namespace_runtime` into `namespace_runtimejob`, `namespace_runtimebatch` will make this config obsolete?